### PR TITLE
Remove years from copyright notes.

### PR DESCRIPTION
**Pull request by vivekvpandya**

### CODE REVIEW

**Comment by bkchr** [comment]
Hey thank you for the pr. Could you please remove the years entirely? We discussed this with the lawyers, but don't yet have done the change.

**Comment by vivekvpandya** [comment]
> Hey thank you for the pr. Could you please remove the years entirely? We discussed this with the lawyers, but don't yet have done the change.

Oh sorry, I will update the PR as per your suggestion soon.

File: bin/node/bench/src/common.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // This file is part of Substrate.
 
-// Copyright (C) 2020-2022 Parity Technologies (UK) Ltd.
+// Copyright (C) Parity Technologies (UK) Ltd.
```
Comment (ggwpez) [inline bin/node/bench/src/common.rs:3]: I see 17 about more files with `\d+\s*Parity Technologies`.

Comment (vivekvpandya) [reply bin/node/bench/src/common.rs:3]: Tried to fix in https://github.com/paritytech/substrate/pull/13415/commits/8d8117269207f59ef639dd1058a4179c444c8678

**Review by ggwpez — APPROVED** [review]
We can probably get rid of `.maintain/update-copyright.sh` then.

**Comment by ggwpez** [comment]
There is one typo on `primitives/npos-elections/src/lib.rs` the SPDX should be in a new line, like in the other files.

**Review by kianenigma — APPROVED** [review]

**Review by shawntabrizi — APPROVED** [review]

**Comment by vivekvpandya** [comment]
Kindly someone merge this.

**Comment by ggwpez** [comment]
bot merge

**Pull request merged**